### PR TITLE
*: refine member/tso/adapter logs

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Part of #2704.

### What is changed and how it works?

This PR refines logs in `member/tso/adapter` packages

### Check List

Tests

- Unit test

Related changes

- Need to cherry-pick to the release branch

### CODE REVIEW

File: pkg/errs/errs.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 PingCAP, Inc.
```
Comment (lhy1024) [inline pkg/errs/errs.go:1]: This file does not seem to include previous error types

Comment (rleungx) [reply pkg/errs/errs.go:1]: > This file does not seem to include previous error types

Yes. I think maybe define error by the package is a better choice?

Comment (ZenoTan) [reply pkg/errs/errs.go:1]: Maybe we should deal with this error file together at last.

Comment (rleungx) [reply pkg/errs/errs.go:1]: Of course

**Pull request merged**